python/pylibraft/pylibraft/cpp/kmeans.pxd

```diff
@@ -33,6 +33,9 @@ from pylibraft.cpp.optional cimport optional
 cdef extern from "raft_distance/kmeans.hpp" \
         namespace "raft::cluster::kmeans::runtime":
 
+    ctypedef const float const_float
+    ctypedef const double const_double
```

Comment:
do we need these typedefs here?   It doesn't seem like they are being used

Reply:
Nope, those need to be removed